devito/ir/clusters/algorithms.py

```diff
@@ -17,10 +17,12 @@
 __all__ = ['clusterize']
 
 
-def clusterize(exprs):
+def clusterize(exprs, **kwargs):
```

Comment:
options=None? Or do you plan to add more kwargs?

Reply:
the caller passes in all of the `kwargs` directly, so this way makes it work seamlessly